packages/scripts/src/application.ts

```diff
@@ -209,6 +278,14 @@ export class Application {
                 createConfigMiddleware(config),
             ]);
         }
+
+        if (serveExternalFeaturesPath) {
+            app.use(`/${EXTERNAL_FEATURES_BASE_URI}`, express.static(resolvedExternalFeaturesPath));
+        }
+
+        app.use('/external', (_, res) => {
```

Comment:
Serving externals only in `run` since this is the only place where we need to get them dynamically